src/lib/guesstimator/formatter/formatters/DistributionTextUpTo.js

```diff
@@ -8,9 +8,12 @@ export const item = Object.assign(
     _symbols: ['->', ':'],
     _numbers(text) { return this._splitNumbersAt(text, this._relevantSymbol(text)) },
     format(g) {
-      const guesstimateType = this.guesstimateType(g)
       const [low, high] = this._numbers(g.text)
-      return {guesstimateType, low, high }
+      if (!(isNaN(low) || isNaN(high))) {
```

Comment:
The validation code doesn't really seem to stop a lot of the functionality of these pieces right now; invalid input will still pre-define your guesstimate type for example. Will work on making this cleaner, but tbh I think a more general refactor is needed in our validity checking/parsing setup here before this can be really isolated; we basically shouldn't be doing anything with an invalid input.